src/lib/seam/components/common-props.tsx

```diff
@@ -1,8 +1,21 @@
 import type { ComponentType } from 'react'
+import type {
+  AccessCodeError,
+  ConnectedAccountError,
+  DeviceError,
+  SeamWarning,
+} from 'seamapi'
+
+export type SeamCompositeError =
+  | ConnectedAccountError
+  | DeviceError
+  | AccessCodeError
 
 export interface RequiredCommonProps {
   className: string | undefined
   onBack: (() => void) | undefined
+  errorFilter: (error: SeamCompositeError) => boolean
+  warningFilter: (warning: SeamWarning) => boolean
```

Comment:
Can you move `SeamCompositeError` to filter.ts or remove it and just be non-dry for now? I'd prefer not to export this names type in the public api (this entire file is re-exported). Better error / names types will be coming to the types package and if we export this we will be introducing conflicting  / redundant types. (This is a good type, but it should not be owned by this project)

```suggestion
export interface RequiredCommonProps {
  className: string | undefined
  onBack: (() => void) | undefined
  errorFilter: (error:  ConnectedAccountError | DeviceError | AccessCodeError) => boolean
  warningFilter: (warning: SeamWarning) => boolean
  
```